mmaction/datasets/pipelines/augmentations.py

```diff
@@ -1190,17 +1665,20 @@ def __call__(self, results):
         else:
             out = imgs
 
-        # Add PCA based noise
-        alpha = np.random.normal(0, self.alpha_std, size=(3, ))
-        rgb = np.array(
-            np.dot(self.eig_vec * alpha, self.eig_val), dtype=np.float32)
-        rgb = rgb[None, None, ...]
+        if self.apply_pca_noise:
+            # Add PCA based noise
+            alpha = np.random.normal(0, self.alpha_std, size=(3, ))
+            rgb = np.array(
+                np.dot(self.eig_vec * alpha, self.eig_val), dtype=np.float32)
```

Comment:
okay this augmentation convert data into float32.
not sure if this is necessary